### PR TITLE
finish single column sort which fully loaded

### PR DESCRIPTION
**Pull request by Cuiyansong**

### CODE REVIEW

File: addon/models/grouping-row-proxy.js [hunk 1]
```diff
@@ -28,5 +28,12 @@ export default Ember.ObjectProxy.extend({
       parentQuery: this.get('selfQuery')
     });
     return lazyArray;
-  }).property()
+  }).property(),
+
+  sort: function (callback) {
+    var children = this.get('children');
+    if(!!children) {
```
Comment (jefflage) [inline addon/models/grouping-row-proxy.js]: why `!!children` instead of just `children`?

Comment (Cuiyansong) [inline addon/models/grouping-row-proxy.js]: Thanks your advice.  :)

Comment (Cuiyansong) [inline addon/models/grouping-row-proxy.js]: Later, I will change it without '!!'.

**Comment by Cuiyansong** [comment]
This PR will be replaced by #48 .

**Pull request closed without merging**